docs/contributing/how-to-release.md

```diff
@@ -67,7 +62,7 @@ The following files need to be uploaded:
     wxWidgets-x.y.z.7z
     wxWidgets-x.y.z.tar.bz2
     wxWidgets-x.y.z.zip
-    wxWidgets-x.y.z-docs-chm.zip
+    wxWidgets-x.y.z-docs.chm
```

Comment:
I know we've traditionally zipped up the CHM manual, but it feels really pointless since it's already mostly compressed, and only saves about 2MB off of a 30MB file. I think we should just stick with the raw CHM here.